docs/20_samples/10_sample-edition/spectra/README.md

```diff
@@ -0,0 +1,13 @@
+# Spectra
+
+This view allows you to visualize spectra related to your sample. There are over 20 different file formats supported for spectra, but only the JCAMP file can be visualized in this view. In order to upload new spectra and convert it in a FAIR format, you should go to the specific tile from the homepage. You can either go back to the homepage and select the tile (e.g. `IR spectra` to upload an infrared spectrum, `isotherm analysis` to upload an isotherm) or click on the colored sticker on the top right of the table. You would be automatically redirected to the tile of interest. 
```

Comment:
@lpatiny the button in the top right corner is probably not clear to many users. Can we/should we add a text "go to app"?